src/site/stages/build/index.js

```diff
@@ -158,6 +167,15 @@ function build(BUILD_OPTIONS) {
   // translating .md files which would allow inPlace() and markdown() to be moved under the
   // permalinks() and navigation() filters making the variable stores uniform between inPlace()
   // and layout().
+
```

Comment:
Are these necessary?

Reply:
Yes. The only alternative I thought of is patching our Metalsmith wrapper (affectionately dubbed "Silversmith" because it's a shiny metal) to output all the files after all the steps, but since there are thousands of files and dozens of steps, that didn't seem like a great approach. (See screenshot below.) Instead, I opted to be deliberate about where we use the plugin.

If you have other suggested approaches, I'm all ears! Er...eyes, as the case may be. :thinking: 

![image](https://user-images.githubusercontent.com/12970166/84953466-65f87f00-b0a8-11ea-80d0-3bf093d6815d.png)

Reply:
:thinking: Giving it more thought, actually, we could introduce another optional parameter to the `smith.use()` command to output the files if `--generate-files` is true. That would work essentially the same.

Reply:
Mostly, I think it's a matter of whether we want to keep this functionality of emitting files in the middle of the pipeline for debugging purposes. If this is a thing we want to keep, it'll need some more polishing. Like better handling of how to specify the output directory, separating those directories per build type, but keeping them distinct from all "real" build outputs, etc. As it stands right now, this is something I whipped up in like 30 minutes to get some examples I could use to show how the build process worked.

Reply:
Let's pause on this while we see how the our [other effort](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10417) is progressing.